app/views/users/edit.html.haml

```diff
@@ -0,0 +1,29 @@
+= content_for :head_content do
```

Comment:
Looks like this view was created by copy & pasting `new_external`. Can we DRY it up some? I think the JS should definitely be shared. I'll leave it up to you to to see where to draw the line for the rest of it.